tests/end2end/User-test.js

```diff
@@ -0,0 +1,182 @@
+import { E2EGlobal } from './helpers/E2EGlobal'
+import { E2EApp } from './helpers/E2EApp'
+import { E2EUser } from './helpers/E2EUser'
+
+
+describe('User Profile/Password editing', function () {
+
+    before("reload page and reset app", function () {
+        E2EApp.resetMyApp(true);
+        E2EApp.launchApp();
+    });
+
+    beforeEach("goto start page and make sure test user is logged in", function () {
+        E2EApp.gotoStartPage();
+        expect(E2EApp.isLoggedIn()).to.be.true;
+    });
+
+
+    it('Buttons Change Password and Edit Profile are not visible for an LDAP user', function () {
+        E2EApp.logoutUser();
+        expect(E2EApp.isLoggedIn()).to.be.false;
+
+        E2EApp.loginLdapUserWithCredentials('ldapUser1', 'ldapPwd1', false);
+        expect(E2EApp.isLoggedIn()).to.be.true;
+
+        if (E2EApp.isLoggedIn()) {
+            browser.click('#navbar-usermenu');
+            E2EGlobal.waitSomeTime();
+            expect(browser.isVisible('#navbar-dlgChangePassword')).to.be.false;
+            expect(browser.isVisible('#navbar-dlgEditProfile')).to.be.false;
+        }
+        browser.click('#navbar-usermenu');
+
+        E2EApp.logoutUser();
+        expect(E2EApp.isLoggedIn()).to.be.false;
+        E2EApp.loginUser();
+    });
+
+    it('User can successefully change his password', function () {
+        if (E2EApp.isLoggedIn()) {
+            let newPassword1 = 'Test12';
+            let newPassword2 = 'Test12';
+            let oldPassword = E2EGlobal.SETTINGS.e2eTestPasswords[0];
+            browser.click('#navbar-usermenu');
+            E2EGlobal.waitSomeTime();
+            browser.click('#navbar-dlgChangePassword');
+            E2EGlobal.waitSomeTime();
+            E2EUser.changePassword(oldPassword, newPassword1, newPassword2);
+            E2EGlobal.waitSomeTime(3000);
+            expect(browser.isVisible('#frmDlgChangePassword')).to.be.false;
+
```

Comment:
I would suggest to add a 

* log out
* log in with new password
* check if login worked

Becaus currently the most important test aspect is missing: did the system really change the password?!

Maybe to ensure that the test does not have side effedts to other tests: change the password back again to the old password. Or other following tests may fail if they try to login user1 with the known password.